### PR TITLE
React Port for mandelbrot

### PR DESCRIPTION
**Pull request by mdfaizan7**
PS ~ Still a WIP.

### CODE REVIEW

File: apps/mandelbrot-react/src/components/FractalImage.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+	useEffect(() => setSize(flexWidth, flexHeight), [flexWidth, flexHeight])
+
+	return (
+		<div {...bind()}>
+			<img
+				unselectable='on'
+				draggable={false}
+				style={{ cursor: 'grab', width: 200 }}
+				src={`http://fractalvalley.net/img?json={%22x%22:${x},%22y%22:${y},%22pix_x%22:${pix_x},%22pix_y%22:${pix_x},%22width%22:256,%22height%22:256,%22max_depth%22:1225,%22test_flags%22:0,%22darken%22:true,%22brighten%22:0,%22modes%22:66,%22colors%22:2,%22texture%22:0,%22edge%22:0,%22var1%22:0,%22var2%22:0,%22renderer%22:%22cpp%22,%22theme%22:0,%22test_vars%22:[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],%22three_d%22:false,%22offset_w%22:0,%22offset_h%22:0,%22eye_sep%22:0,%22eye_adjust%22:0}`}
```
Comment (stevehoover) [inline apps/mandelbrot-react/src/components/FractalImage.js:53]: I think you can delete the %22's. I believe these are just spaces in the JSON, which are non-functional.

File: apps/mandelbrot-react/src/components/Colors.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import OptionCard from './OptionCard'
+import store from '../store'
+
+const colors_array = [
```
Comment (stevehoover) [inline apps/mandelbrot-react/src/components/Colors.js:5]: Maybe it makes sense to define a more generic component that is an EnumerateFractalAttribute and use if for colors, etc.? (I don't have experience w/ React, so just a suggestion.)

**Review by stevehoover — APPROVED** [review]
Looks great! We'll do a more complete review interactively soon.

**Comment by stevehoover** [comment]
I accepted your pull request. Aso, this resulted in 3 dependabot
auto-pull-requests to incorporate security fixes. I accepted these blindly.
You might want to pull from master and npm install to test these updates.

This is definitely helping me to understand your skills. I hope you are
enjoying the project. Busy week coming up for me, but we'll talk soon.

Steve Hoover <https://www.linkedin.com/in/steve-hoover-a44b607/>
774-773-8333

<http://www.redwoodeda.com/>

File: apps/mandelbrot-react/src/components/FractalImage.js (hunk 1 above)
Comment (mdfaizan7) [reply apps/mandelbrot-react/src/components/FractalImage.js:53]: I believe that %22's represent ".  But I do think that we can replace it with ".

**Comment by mdfaizan7** [comment]
Hi Steve, sorry I was not able to reply earlier. I had my mid-sem exams going on at that time and I was pretty busy with that.

**Pull request merged**